hokusai/development.yml

```diff
@@ -1,26 +1,28 @@
 ---
-version: '2'
+version: "2"
 services:
   horizon:
     env_file: ../.env
     depends_on:
-    - horizon-postgres
-    - horizon-redis
+      - horizon-postgres
+      - horizon-redis
     environment:
-    - DATABASE_URL=postgresql://postgres:@horizon-postgres/horizon_development
-    - REDIS_URL=redis://horizon-redis
+      - DATABASE_CLEANER_ALLOW_REMOTE_DATABASE_URL=true
```

Comment:
I don't think this project uses the database_cleaner library so this shouldn't be required.